packages/astro/src/runtime/client/dev-toolbar/apps/audit/ui/audit-ui.ts

```diff
@@ -147,18 +147,32 @@ function buildAuditCard(
 
 	const message = document.createElement('p');
 	message.classList.add('audit-message');
-	message.innerHTML = rule.message;
+	message.innerHTML = simpleRenderMarkdown(rule.message);
 	extendedInfo.appendChild(message);
 
 	const description = rule.description;
 	if (description) {
 		const descriptionElement = document.createElement('p');
 		descriptionElement.classList.add('audit-description');
-		descriptionElement.innerHTML = description;
+		descriptionElement.innerHTML = simpleRenderMarkdown(description);
 		extendedInfo.appendChild(descriptionElement);
 	}
 
 	card.shadowRoot.appendChild(extendedInfo);
 
 	return card;
 }
+
+/**
+ * Render a very small subset of Markdown to HTML or a CLI output
+ */
+function simpleRenderMarkdown(markdown: string) {
+	const linkRegex = /\[([^[]+)\]\((.*)\)/g;
+	const boldRegex = /\*\*(.+)\*\*/g;
+	const codeRegex = /`([^`]+)`/g;
```

Comment:
I would move these regexes outside of this function. We create them only once and not every time we run the function.